<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartInterface</class>
 <widget class="QWidget" name="StartInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="background_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#background_widget
{
	background-image: url(://interface/resource/startinterface/startInterface.jpg);
}</string>
   </property>
   <widget class="QLabel" name="system_version">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>60</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="soft_version">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="startProgress">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>931</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{
	border:none;
}

QSlider::sub-page:horizontal{
background-color:#708EB8;
border-bottom-left-radius: 15;
border-top-left-radius: 15;
border-top:1px solid #DAE0E3;
border-bottom:1px solid #9AAECB;
}

QSlider::add-page:horizontal{
background-color:#ECEEEF;
border-bottom-right-radius: 15;
border-top-right-radius: 15;
border-top:1px solid #DAE0E3;
border-bottom:1px solid #ECEEEF;
}

QSlider::handle:horizontal{
background-color:#708EB8;
width:30;
height:25;
border-radius:15;
}
 

 </string>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
